pkg/protobuf/v2/Makefile

```diff
@@ -40,6 +40,9 @@ artifacts/protobuf/bin/run-protoc: $(PROTOC_COMMAND)
 artifacts/protobuf/bin/protoc-gen-go artifacts/protobuf/bin/protoc-gen-go-grpc: go.mod
 	$(MF_ROOT)/pkg/protobuf/v2/bin/install-protoc-gen-go "$(MF_PROJECT_ROOT)/$(@D)"
 
+artifacts/protobuf/bin/protoc-gen-swift artifacts/protobuf/bin/protoc-gen-grpc-swift:
```

Comment:
Is it actually correct that the Go binary is called `protoc-gen-go-grpc` (with `grpc` on the end) and the Swift one is called `protoc-gen-grpc-swift` (with `grpc` in the middle)? Or is one of them a mistake?

Reply:
Did my own search, and it seems this is indeed the case:
- https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc
- https://github.com/grpc/grpc-swift#versions

Reply:
Good catch! That actually tripped me up for longer than I care to admit because I originally copied from the go and everything worked except grpc generation...